packages/module/patternfly-docs/content/extensions/component-groups/examples/Unavailable/Unavailable.md

```diff
@@ -0,0 +1,25 @@
+---
+# Sidenav top-level section
+# should be the same for all markdown files
+section: extensions
+subsection: Component groups
+# Sidenav secondary level section
+# should be the same for all markdown files
+id: Unavailable
```

Comment:
Is this component name set in stone? If there's wiggle room, I think something like "unavailable page" might help users have a better understanding of the component before clicking in and reading the details

Reply:
The name is not set in stone.  I like unavailable content.